docs/microbit_micropython_api.rst

```diff
@@ -157,63 +214,72 @@ Images API::
     # returns a new image created by multiplying the brightness of each pixel by n
     image * n
 
-    # built-in images.
-    Image.HEART
-    Image.HEART_SMALL
-    Image.HAPPY
-    Image.SMILE
-    Image.SAD
-    Image.CONFUSED
-    Image.ANGRY
-    Image.ASLEEP
-    Image.SURPRISED
-    Image.SILLY
-    Image.FABULOUS
-    Image.MEH
-    Image.YES
-    Image.NO
-    Image.CLOCK12 # clock at 12 o' clock
-    Image.CLOCK11
-    ... # many clocks (Image.CLOCKn)
-    Image.CLOCK1 # clock at 1 o'clock
-    Image.ARROW_N
-    ... # arrows pointing N, NE, E, SE, S, SW, W, NW (microbit.Image.ARROW_direction)
-    Image.ARROW_NW
-    Image.TRIANGLE
-    Image.TRIANGLE_LEFT
-    Image.CHESSBOARD
-    Image.DIAMOND
-    Image.DIAMOND_SMALL
-    Image.SQUARE
-    Image.SQUARE_SMALL
-    Image.RABBIT
-    Image.COW
-    Image.MUSIC_CROTCHET
-    Image.MUSIC_QUAVER
-    Image.MUSIC_QUAVERS
-    Image.PITCHFORK
-    Image.XMAS
-    Image.PACMAN
-    Image.TARGET
-    Image.TSHIRT
-    Image.ROLLERSKATE
-    Image.DUCK
-    Image.HOUSE
-    Image.TORTOISE
-    Image.BUTTERFLY
-    Image.STICKFIGURE
-    Image.GHOST
-    Image.SWORD
-    Image.GIRAFFE
-    Image.SKULL
-    Image.UMBRELLA
-    Image.SNAKE
-    # built-in lists - useful for animations, e.g. display.show(Image.ALL_CLOCKS)
-    Image.ALL_CLOCKS
-    Image.ALL_ARROWS
-
-The accelerometer
------------------
+**Built-in images**
+
+``Image.HEART``
+``Image.HEART_SMALL``
+``Image.HAPPY``
+``Image.SMILE``
+``Image.SAD``
+``Image.CONFUSED``
+``Image.ANGRY``
+``Image.ASLEEP``
+``Image.SURPRISED``
+``Image.SILLY``
+``Image.FABULOUS``
+``Image.MEH``
+``Image.YES``
+``Image.NO``
+``Image.TRIANGLE``
+``Image.TRIANGLE_LEFT``
+``Image.CHESSBOARD``
+``Image.DIAMOND``
+``Image.DIAMOND_SMALL``
+``Image.SQUARE``
+``Image.SQUARE_SMALL``
+``Image.RABBIT``
+``Image.COW``
+``Image.MUSIC_CROTCHET``
+``Image.MUSIC_QUAVER``
+``Image.MUSIC_QUAVERS``
+``Image.PITCHFORK``
+``Image.XMAS``
+``Image.PACMAN``
+``Image.TARGET``
+``Image.TSHIRT``
+``Image.ROLLERSKATE``
+``Image.DUCK``
+``Image.HOUSE``
+``Image.TORTOISE``
+``Image.BUTTERFLY``
+``Image.STICKFIGURE``
+``Image.GHOST``
+``Image.SWORD``
+``Image.GIRAFFE``
+``Image.SKULL``
+``Image.UMBRELLA``
+``Image.SNAKE``
+
+Clock:
+
+``Image.CLOCK1`` ``Image.CLOCK2`` ``Image.CLOCK3`` ``Image.CLOCK4``
+``Image.CLOCK5`` ``Image.CLOCK6`` ``Image.CLOCK7`` ``Image.CLOCK8``
+``Image.CLOCK9`` ``Image.CLOCK10`` ``Image.CLOCK11`` ``Image.CLOCK12``
+
+Arrows:
+
+``Image.ARROW_N`` ``Image.ARROW_NE`` ``Image.ARROW_E`` ``Image.ARROW_SE``
+``Image.ARROW_S`` ``Image.ARROW_SW`` ``Image.ARROW_W`` ``Image.ARROW_NW``
+
+Built-in image lists:
+
+These will display an animation.
```

Comment:
The new image list looks great!

What do you think about adding a bit more info like this for this section?



```suggestion
The following are Python lists of images, useful for automatically displaying an
animation or manually iterating through them.
```

I was also thinking about removing the title to reduce a bit the large blank spaces between arrows and lists?:
![image](https://user-images.githubusercontent.com/29712657/100907596-6ce24500-34c2-11eb-8693-4dfe3498f27b.png)